packages/tests/package.json

```
@@ -5,15 +5,19 @@
   "sideEffect": false,
   "type": "module",
   "scripts": {
-    "test": "playwright test",
-    "test:ui": "playwright test --ui",
+    "test": "NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test",
+    "test:ui": "NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test --ui",
     "lint": "eslint ."
```

Suggestion: We can ensure browser engines are installed before launching test suite.

```suggestion
    "test": "playwright install && NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test",
    "test:ui": "playwright install && NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test --ui",
    "lint": "eslint ."
```

Or we can simply add an install script to assist devs on manual install

```suggestion
    "test:install": "playwright install"
    "test": "NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test",
    "test:ui": "NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test --ui",
    "lint": "eslint ."
```

---

I like it more with the specific command for install, don't want playwright to try install each time we want to run tests xD